plugins/CorePluginsAdmin/vue/src/PluginSettings/PluginSettings.vue

```diff
@@ -227,7 +227,7 @@ export default defineComponent({
           context: 'success',
           type: 'transient',
         });
-        NotificationsStore.scrollToNotification('generalSettings');
```

Comment:
w/ the new code the way to do this is:

```
const notificationInstanceId = NotificationsStore.show({
  message: translate('CoreAdminHome_PluginSettingsSaveSuccess'),
  id: 'generalSettings',
  context: 'success',
  type: 'transient',
});
NotificationsStore.scrollToNotification(notificationInstanceId);
```

The reason for this change is because Notification.vue makes an AJAX request in markNotificationAsRead() if an ID is specified, but that isn't needed for notifications added client side only. So if you want to avoid making that request, you don't have to specify the `id:` property.

Reply:
I'll add documentation for this later unless it's desired to revert this to the old behavior.

cc @sgiehl

Reply:
@diosmosis that makes more sense. updating it now